settings.py

```diff
@@ -82,3 +82,7 @@
 PR_STALE_HOURS = 36
 
 API_COOLDOWN_RESET_PADDING = 30
+
+# list of autocracy members
+with open("data/autocracy.txt", "r") as f:
```

Comment:
Maybe it's a common practice and i don't know about it, but what's the point of using file for this? why not a list in `settings.py` for example?